doc/source/admin/production.md

```diff
@@ -152,6 +152,26 @@ To get started with setting up local data, please see [Data Integration](https:/
 
 File sizes have grown very large thanks to rapidly advancing sequencer technology, and it is not always practical to upload these files through the browser. Thankfully, a simple solution is to allow Galaxy users to upload them via FTP and import those files in to their histories. Configuration for FTP is explained on the [File Upload via FTP](special_topics/ftp.md) page.
 
+### Protect Galaxy against data loss due to misbehaving tools
+
+Tools have access to the paths of input and output data sets which are stored in
+``file_path`` and by default the credentials used for running tools are the same
+as for running Galaxy. Thus its possible that tools modify data in Galaxy's
+``file_path``. Examples for such changes are:
+
+- Addition of additional files, e.g. indices, which is a problem for cleaning up data, because Galaxy does not know about these files.
```

Comment:
```suggestion
- Creation of additional files, e.g. indices, which is a problem for cleaning up data, because Galaxy does not know about these files.
```